packages/paste-website/src/components/AspectRatio.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import {Absolute} from '@twilio-paste/absolute';
+import {Box} from '@twilio-paste/box';
+
+interface AspectRatioProps {
+  aspectRatio: string;
+  children: React.ReactElement;
+}
+
+const RATIO_REGEX = /^(\d+:\d*)$/;
+
+const isCorrectPattern = (ratio: string): boolean => RATIO_REGEX.test(ratio);
+
+const handlePropValidation = ({aspectRatio, children}: AspectRatioProps): void => {
+  const hasRatio = aspectRatio != null && aspectRatio !== '';
+
+  if (!hasRatio) {
+    throw new Error(`[Paste: AspectRatio] Missing hasRatio prop for AspectRatio.`);
```

Comment:
`[Paste: AspectRatio] Missing 'aspectRatio' prop.`

Reply:
Changed this as well.